File: source/api/disputes/update/_update.rb
require "omise"

Omise.api_key = "skey_test_4xs8breq3htbkj03d2x"

dispute = Omise::Dispute.retrieve("dspt_test_4zgf15h89w8t775kcm8")
dispute.update({
  message: "Proofs and other information regarding the disputed charge ..."
})

File: source/api/disputes/update/_update.java
import java.util.HashMap;

import co.omise.model.Dispute;

Dispute dispute = Dispute.retrieve("dspt_test_4zgf15h89w8t775kcm8");
dispute.update(new HashMap<String, Object>() {
  {put("message", "Proofs and other information regarding the disputed charge ...");}
});
